pkgs/development/python-modules/tiny-cuda-nn/default.nix

```diff
@@ -0,0 +1,103 @@
+{
+  buildPythonPackage,
+  cmake,
+  fetchFromGitHub,
+  lib,
+  ninja,
+  pybind11,
+  symlinkJoin,
+  torch,
+  which,
+}:
+assert lib.asserts.assertMsg
+(torch.cudaSupport)
+"tiny-cuda-nn requires torch to be built with cudaSupport"; let
+  inherit (lib) lists strings;
+  inherit (torch) cudaPackages cudaSupport;
+  inherit (cudaPackages) cudatoolkit cudaFlags;
+
+  cuda-redist = symlinkJoin {
+    name = "cuda-redist";
+    paths = with cudaPackages; [
+      cuda_cudart
+      cuda_nvcc
+      libcublas
+      libcusparse
+      libcusolver
+    ];
+  };
+
+  dotlessSemicolonCudaCapabilities = let
+    dropPTX = builtins.filter (c: ! strings.hasSuffix "PTX" c) cudaFlags.cudaCapabilities;
+    dropDot = lists.map (strings.replaceStrings ["."] [""]) dropPTX;
+  in
+    strings.concatStringsSep ";" (lists.unique dropDot);
+
+  name = "tiny-cuda-nn";
+  version = "1.6";
+in
+  buildPythonPackage {
+    inherit name version;
+    format = "setuptools";
+
+    src = fetchFromGitHub {
+      owner = "NVlabs";
+      repo = name;
+      rev = "v${version}";
+      fetchSubmodules = true;
+      hash = "sha256-qW6Fk2GB71fvZSsfu+mykabSxEKvaikZ/pQQZUycOy0=";
+    };
+
+    sourceRoot = "source/bindings/torch";
```

Comment:
Would be nice to build the python-free library as well, we could re-use that e.g. for https://github.com/NVlabs/instant-ngp/blob/master/CMakeLists.txt

...but it seems `tiny-cuda-nn` doesn't have any `install()` targets yet anyway

Reply:
Originally I had done something like that and intended to use the toPythonModule command but when I saw there were no install targets I gave up.

In that case, should I make a custom install phase? Or is it enough to set `dontInstall = true;` and the derivation will have just the built libraries?